facilities/migrations/0001_initial.py

```diff
@@ -199,12 +226,12 @@ class Migration(migrations.Migration):
                 ('sub_division', models.CharField(help_text=b'This is a further division of the facility type e.g Hospitals can be further divided into District hispitals and Provincial Hospitals.', max_length=100, null=True, blank=True)),
                 ('created_by', models.ForeignKey(related_name='+', on_delete=django.db.models.deletion.PROTECT, default=common.models.base.get_default_system_user_id, to=settings.AUTH_USER_MODEL)),
                 ('preceding', models.ForeignKey(related_name='preceding_type', blank=True, to='facilities.FacilityType', help_text=b'The facility type that comes before this type', null=True)),
-                ('suceedding', models.ForeignKey(related_name='suceedding_type', blank=True, to='facilities.FacilityType', help_text=b'The facility type that comes afetr this type', null=True)),
```

Comment:
This was removed a long time ago. I encountered a cyclic dependency while creating data for the model.